packages/@expo/cli/src/export/embed/exportEmbedAsync.ts

```diff
@@ -64,6 +64,13 @@ function guessCopiedAppleBundlePath(bundleOutput: string) {
 }
 
 export async function exportEmbedAsync(projectRoot: string, options: Options) {
+  // The React Native build scripts always enable the cache reset but we shouldn't need this in CI environments.
+  // By disabling it, we can eagerly bundle code before the build and reuse the cached artifacts in subsequent builds.
+  if (env.CI && options.resetCache) {
+    debug('CI environment detected, disabling reset cache');
```

Comment:
```suggestion
    debug('CI environment detected, disabling automatic cache reset');
```

Might need to emphasize the "default value" or "automatic" cache reset (as in, `--reset-cache` still works)